spec/save_operation_spec.cr

```diff
@@ -32,6 +32,11 @@ end
 
 private class SaveLineItem < LineItem::SaveOperation
   permit_columns :name
+  before_save prepare
+
+  def prepare
+    1
```

Comment:
This doesn't have to do anything. It was just to test that before the change, these specs failed when this was here. Now it passes.